browser/resources/settings/email_aliases_page/tests/email_aliases.test.tsx

```diff
@@ -540,4 +543,28 @@ describe('ManagePageConnected', () => {
       expect(screen.queryByText('alias2@brave.com')).not.toBeInTheDocument()
     })
   })
+
+  it('shows error message when auth fails', async () => {
+    const mockEmailAliasesService = new MockEmailAliasesService()
+
+    await act(async () => {
+      render(<ManagePageConnected
+        emailAliasesService={mockEmailAliasesService}
+        bindObserver={createBindObserver(mockEmailAliasesService)}
+      />)
+    })
+
+    await act(() => {
+      mockEmailAliasesService.notifyObserverAuthStateChanged(
+        AuthenticationStatus.kAuthenticating,
+        'test@brave.com',
+        'mockErrorMessage'
+      )
+    })
+
+    await waitFor(() => {
+      expect(screen.getByText(/mockErrorMessage/)).toBeInTheDocument()
```

Comment:
we should also test:

1. Aliases disappear if you're suddenly unauthenticated
2. Data doesn't persist across different logins
3. You can recover from an error state (i.e. if auth succeeds after failing)
4. Empty string error message?

Reply:
I added tests for these